### ISSUE TITLE
Accessibility for selects, dropdowns, switch etc

### ISSUE THREAD

**Issue by MartinMuzatko** [myliang/fish-ui#46]
Currently, the elements are not able to be focused, toggled or changed with the keyboard.

All events should be able to be focused using `tab`
- selects,dropdowns should support `up`/`down`/`enter` and any letter key (a,b,c,d,...) to search and highlight items
- switches should support `space` to change value

It might be helpful to use a component such as `vue-global-events`.

Otherwise this lib looks promising. Minimalistic but I am missing a few essential accessibility features.

**Comment by wysisoft** [comment]
ya i like the ui but keyboard is critical, compare your stuff to vuetifykeyboard access

**Comment by MartinMuzatko** [comment]
Btw, this is not a question, this is a feature request.

**Comment by juanri0s** [comment]
Yeah this might be the only thing hurting the library right now. Definitely think accessibility needs attention across all elements. I can't tab onto a button, even with hacks.
